src/include/daos_srv/vos_types.h

```diff
@@ -330,8 +330,10 @@ enum {
 	VOS_IT_FOR_DISCARD	= (1 << 8),
 	/** Entry is not committed */
 	VOS_IT_UNCOMMITTED	= (1 << 9),
+	/* Iterate all active DTX entries. */
+	VOS_IT_LIST_ACT_DTX	= (1 << 10),
```

Comment:
What's the purpose of this flag? Could we use existing flag like IT_FOR_PURGE instead of introducing a new one?

Also, I don't quite see the purpose of VOS_IT_CLEANUP_DTX.

Reply:
IT_FOR_PURGE is used by aggregation when iterate objects/keys that may touch some DTX entries, but IT_LIST_ACT_DTX is used for iterating DTX entries themselves, they are quite different. Anyway, I will consider to merge IT_CLEANUP_DTX and IT_LIST_ACT_DTX.